auto_nag/scripts/not_landed.py

```diff
@@ -148,7 +148,15 @@ def check_splinter(self, attachment):
         if attachment["is_patch"] == 0 or attachment["is_obsolete"] == 1:
             return None
 
-        data = base64.b64decode(attachment["data"]).decode("utf-8")
+        data = base64.b64decode(attachment["data"])
+        try:
+            data = data.decode("utf-8")
+        except UnicodeDecodeError:
+            try:
+                data = data.decode("iso-8859-1")
+            except UnicodeDecodeError:
+                return None
```

Comment:
At this point, you could do `data = data.decode("utf-8", "ignore")`.

Or even, since you're just checking if the data starts with "https://github.com", you could avoid the decoding completely and do:
```
if data.startswith(b"https://github.com"):
    return None
```

Reply:
oups, sorry